### PR TITLE
[Doc Improvement][Build bots for Teams - Set different bot name to display in chat window][3676872]

### PR DESCRIPTION
**Pull request by v-vanv**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0796d4101aff7d17b1c9bfb6a600b4b29663bbaa/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 91 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0694daf7-5e22-43e6-a51e-ab645610db2f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ad974bcdc012368dd2a5b0bd351572a37b9086c8/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80bb21ce-4245-4d22-9097-ef91740d30ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/220efca7dc22fcbff48adae94ab2b21d3fc2da44/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/abe36a34-581e-46c7-bd0d-0f692e2f3acb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0cfe41849a461eaa538b10834be6a05ec8059cf5/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf170083-5f37-4e05-9b2e-777cc06e18b7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2d468480425411bed5a4545f46af9361557260a2/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/676e4118-fed7-4146-a4bc-a4ec4a9bb68a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08b6500771e688946eda604fe36e71bf37c7a4b1/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 89 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0fb01205-7e74-495f-afdb-60915b22a3b8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6f81bb868de1df167d85021d3e8c49b4fa9cde99/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46dbca93-6ecc-4f51-8e15-425d8dd753f9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Prasad-MSFT — APPROVED** [review]
Changes look great. Approving the PR

File: msteams-platform/bots/what-are-bots.md [hunk 1]
```diff
@@ -63,6 +63,14 @@ You can add single sign-on authentication to your conversation bot using the fol
 
 Bot configuration experience helps the users to interact with the bot in Teams. Users can interact with the bot either by sending a message or selecting a command from the command list. After the bot is installed in a channel or team, all the members from the channel or team can provide inputs to the bot at the same time, the bot only considers the last input provided by the user. For more information, see [bot configuration experience](how-to/bot-configuration-experience.md).
 
+You can change the bot name displayed in Teams environment and you need to update it in the following occurrences:
+
+* The bot name displayed as a title in the chat window managed within your app manifest. You can update this bot name directly in the app manifest.
+
+* The bot name displayed in your Teams chat list and the bot name appears with each message in the chat pane are managed within the Azure portal. You can update bot name in the **Bot profile** page in the Azure portal.
```
Comment (Wajeed-msft) [inline msteams-platform/bots/what-are-bots.md:70]:
```suggestion
* The bot name displayed in your Teams chat list and the bot name appears with each message in the chat pane are managed within the Azure portal. You can update bot name in the **Bot profile** page in the Azure portal for sideloaded application.
```

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/147556b90f706373076cf195597fea21bad77f73/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d709e568-1389-4862-97fe-33ce9464bfe9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/what-are-bots.md (hunk 1 above)
Comment (v-vanv) [reply msteams-platform/bots/what-are-bots.md]: Updated

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f88e1c3da58e64290938530f50ad21e09015f3f5/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a0aeaa53-3ca3-4b46-a21b-357152131626) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d505bbda5b9ce5d833394b0789ee733b01362394/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c0629bd2-56eb-4550-81ac-daac455106cb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/deda85b1772a331de847793dd7def2188f2b07da/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/565b0057-6baa-4a2a-bb74-d4bd10ff7f21) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1c25579c8b7b26b5062b1064fa7fa6a613552adb/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 90 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4a96901f-58d6-4759-831b-63dd60e326c4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**